builders/integrations/indexers/moralis.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Moralis
+description: Learn how to build on the Moonbeam network with Moralis' API suite.
+---
+
+# Index Moonbeam with Moralis
+
+## Introduction {: #introduction }
+
+As a one-stop solution for blockchain development, [Moralis](https://moralis.io/) offers a comprehensive platform that empowers developers to create, launch, and scale decentralized applications (dApps) with ease. This guide will show you how to access the API endpoints for Moonbeam using curl commands and JavaScript and Python snippets.
+
+Moralis' offerings are structured into four primary categories of API services-
```

Comment:
```suggestion
Moralis' offerings are structured into four primary categories of API services:
```

Reply:
Understood. I have undid those changes. It is all good!